### PR TITLE
chore(client/executor): remove unused error InvalidCode and improve docs

### PR DESCRIPTION
**Pull request by yjhmelody**

### CODE REVIEW

File: client/executor/src/wasm_runtime.rs [hunk 1]
```diff
@@ -201,16 +199,14 @@ impl RuntimeCache {
...
-	///
 	/// `f` - Function to execute.
 	///
 	/// `H` - A compile-time list of host functions to expose to the runtime.
 	///
 	/// # Returns result of `f` wrapped in an additional result.
 	/// In case of failure one of two errors can be returned:
 	///
-	/// `Err::InvalidCode` is returned for runtime code issues.
+	/// `Err::RuntimeConstruction` is returned for runtime construction issues.
```
Comment (kianenigma) [inline client/executor/src/wasm_runtime.rs:209]: where is this error raised from? I don't see it in this function.

**Review by kianenigma — APPROVED** [review]
LGTM

**Review by bkchr — APPROVED** [review]

File: client/executor/src/wasm_runtime.rs (hunk 1 above)
Comment (bkchr) [reply client/executor/src/wasm_runtime.rs:209]: Good point, this error also doesn't seem to be used anywhere.

Comment (yjhmelody) [reply client/executor/src/wasm_runtime.rs:209]: #[error(transparent)]
RuntimeConstruction(#[from] WasmError),

**Comment by bkchr** [comment]
@yjhmelody the CI is not happy.

**Comment by yjhmelody** [comment]
@bkchr It's happy now ?

**Comment by yjhmelody** [comment]
@bkchr  Can we merge it ?

**Review by shawntabrizi — APPROVED** [review]

**Pull request merged**